src/main/webapp/Maps/user-map.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>User Map</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBVU50aOCT7FRl8BZppVuXuZ05Wa41O0M4"></script>
+  <script>
+
+    let map;
+
+    function createMap(){
+   
+      map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: 38.5949, lng: -94.8923},
+        zoom: 4
+      });
+   
+     map.addListener('click', (event) => {
+      new google.maps.Marker({
+        position: event.latLng,
+        map: map
+      });  
+});
+    }
+  </script>
+  <style>
+    #map{
+      width: 500px;
+      height: 500px;
+    }
+  </style>
+</head>
+<body onload="createMap();">
```

Comment:
nit: but a newline between head and body. Make sure all tabs are 2 spaces (check body!)